packages/e2e/features/ui/components/authenticator/sign-in-with-multi-alias.feature

```diff
@@ -10,25 +10,25 @@ Feature: Sign In with Multiple Aliases
   Background:
     Given I'm at the sign in page
 
-  @react
+  @next @react
```

Comment:
These aren't Angular or Vue because, I assume, it hasn't been implemented there yet?

Reply:
I actually addressed this in my improved credentials pr. The functionality is there for both frameworks, we just needed to set up a proper example that uses the multi-alias backend.